openedx/features/partners/tests/test_reset_password.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+from openedx.features.partners.tests.factories import PartnerUserFactory, PartnerFactory
+from rest_framework.test import APITestCase
+from rest_framework import status
+from student.tests.factories import UserFactory
+
+
+class ResetPasswordTestCases(APITestCase):
+
+    def setUp(self):
+        self.end_point = reverse('partner_reset_password')
+
+    def test_valid_partner_email(self):
+        valid_data = {
```

Comment:
Lets add documentation for all the test cases, stating test case and how data is generated

Reply:
Done.